AndelaOSP/art-dashboard
# #162694165 Fix email validation error display

## What does this PR do?

Fixes an issue where an error was displayed under the email input field even when the email was valid.

## Description of Task to be completed?

Same as above

## How should this be manually tested?

* Log in
* Click on the `Users` link
* Navigate to the `Security Users` tab
* Click on the `Add Security User` button
* Enter an email in the email input field and observe behavior

## What are the relevant pivotal tracker stories?

[162694165](https://www.pivotaltracker.com/story/show/162694165)

## Any background context you want to add?

N/A

## Important notes

N/A

## Packages installed

N/A

## Deployment note

N/A

## Related PRs branch | PR (branch_name) | (pr_link)

Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()

## Todos


## Screenshots

![screen recording 2019-01-08 at 05 36 pm](https://user-images.githubusercontent.com/31339738/50838698-b1651480-136f-11e9-892b-074da110e5c1.gif)
